app/assets/src/components/common/DetailsSidebar/GeneDetailsMode/GeneDetailsMode.jsx

```diff
@@ -0,0 +1,430 @@
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {ontology.label}
+        </a>, which is released under the{" "}
+        <a
+          href="https://creativecommons.org/licenses/by/4.0/"
+          className={cs.cardLink}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Creative Commons CC-BY license version 4.0
+        </a>{" "}
+        by McMaster University.
+      </div>
+    );
+  }
+
+  renderNotFound() {
+    const { geneName } = this.props;
+    return <div className={cs.cardLicense}>No data found for {geneName}</div>;
+  }
+
+  renderHeader() {
+    const { loading, ontology } = this.state;
+    const { geneName } = this.props;
+    if (loading) {
+      return <div className={cs.loadingMsg}>Loading...</div>;
+    }
+    return (
+      <div className={cs.title}>
+        {ontology.label !== "---" ? ontology.label : geneName}
+      </div>
+    );
+  }
+
+  renderOntology() {
+    const { ontology, collapseOntology } = this.state;
+    return (
+      <div>
+        <div className={cs.text}>
+          <div className={cs.textInner}>
+            {CARD_SYNONYMS}: <em>{ontology.synonyms}</em>
+          </div>
+        </div>
+        <div className={cs.subtitle}>Description</div>
+        <div className={cs.text}>
+          <div className={cs.textInner}>{ontology.description}</div>
+        </div>
+        {!collapseOntology && (
+          <div>
+            <div className={cs.subtitle}>{CARD_FAMILY}</div>
+            <div className={cs.text}>
+              <div className={cs.textInner}>{ontology.geneFamily}</div>
+            </div>
+            <div className={cs.subtitle}>{CARD_CLASS}</div>
+            <div className={cs.text}>
+              <div className={cs.textInner}>{ontology.drugClass}</div>
+            </div>
+            <div className={cs.subtitle}>{CARD_MECHANISM}</div>
+            <div className={cs.text}>
+              <div className={cs.textInner}>{ontology.resistanceMechanism}</div>
+            </div>
+            <div className={cs.subtitle}>{CARD_PUBLICATIONS}</div>
+            <div className={cs.text}>{this.renderPublications()}</div>
+          </div>
+        )}
+        {collapseOntology && (
+          <div className={cs.expandLink} onClick={this.expandOntology}>
+            Show More
+          </div>
+        )}
+        <div className={cs.text}>
+          <div className={cs.textInner}>{this.renderCARDLicense()}</div>
+        </div>
+      </div>
+    );
+  }
+
+  renderPublications() {
+    const { ontology } = this.state;
+    return ontology.publications.map(publication => {
+      const citation = /.*(?=(\(PMID))/.exec(publication)[0];
+      const pmidText = /(PMID)\s[0-9]*/.exec(publication)[0];
+      const pubmedId = pmidText.split(" ")[1];
+      return (
+        <div className={cs.textInner} key={pubmedId}>
+          {citation}
+          <div className={cs.link}>
+            (<a
```

Comment:
you probably want to add another `span` in here because the entire row turns into the pointer cursor

Reply:
also maybe the parentheses shouldn't be part of the link

Reply:
Fixed!